src/main/java/seedu/address/model/wedding/Wedding.java

```diff
@@ -0,0 +1,89 @@
+package seedu.address.model.wedding;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+
+import seedu.address.commons.util.ToStringBuilder;
+
+/**
+ * Represents a wedding plan.
+ */
+public class Wedding {
+    private Husband husband;
+    private Wife wife;
+    private Date date;
+    private Venue venue;
+    private ContactMap contactList;
```

Comment:
I don't think ContactMap is initalised before use, may result in null pointer exception.